genanki/model.py

```diff
@@ -4,13 +4,30 @@
 import yaml
 
 class Model:
-  def __init__(self, model_id=None, name=None, fields=None, templates=None, css=''):
+  def __init__(self, model_id=None, name=None, fields=None, templates=None, css='',
+          latex_header=None, latex_footer=None):
```

Comment:
Instead of defaulting to `None`, you can default to `_DEFAULT_LATEX_HEADER` (see below) and then you don't need the `if latex_header is None` logic.